src/UI/App.hs

```diff
@@ -146,9 +148,9 @@ initialState conf =
          (L.list ListOfFiles mempty 1)
          (E.editor ManageFileBrowserSearchPath Nothing path)
     mailboxes = view (confComposeView . cvIdentities) conf
-    s = AppState conf mi mv (initialCompose mailboxes) Nothing viewsettings fb
-  in
-    applySearch s
+    epoch = UTCTime (fromGregorian 2018 07 18) 1
```

Comment:
Heh, what epoch is this?  (Not suggesting to change it... just curious :) )

Reply:
First commit :)

Reply:
Off by one year error :D  It was 2017.  (Don't worry about it).